modules/gdscript/tests/scripts/analyzer/features/preload_constant_types_are_inferred.gd

```diff
@@ -0,0 +1,5 @@
+const Constants = preload("gdscript_to_preload.gd")
+
+func test():
+    var a := Constants.A
+    print(a)
```

Comment:
Missing newline at EOF.

Reply:
Newline added.

Reply:
Sorry, I didn't do the amend correctly, now it is added.

Reply:
@akien-mga somehow the last commit amend ignored the newline I added to the file. I had to add an additional newline to have it affect the commit. Never encountered something like this...
Now the file has an additional newline at the end of the file :)